Bio/GenBank/Scanner.py

```diff
@@ -352,7 +356,7 @@ def parse_footer(self):
             if line == "//":
                 break
         self.line = line
-        return ([], "")  # Dummy values!
+        return [[], ""]  # Dummy values!
```

Comment:
This has changed the return type from a tuple to a list? How about ``return [], ""`` instead?